### DISCUSSION TITLE
Dear all

### DISCUSSION THREAD

**Discussion by Spartiate91** [community/community#37257] [New to GitHub]
New to GitHub, i am here to learn. I'm a noob in this universe.

Kindly regards,
Johann.

**Comment by radar-blip** [comment]
Hi @Spartiate91! Welcome to the GitHub Universe!!

**Reply by Spartiate91** [reply]
Thanks a lot 😃

**Comment by queenofcorgis** [comment]
Welcome @Spartiate91 - we're glad to have you here, you're certainly in the right place 🌟 ! Are you looking to learn anything specific? If not, can I recommend starting with our [New User Checklist](https://github.com/orgs/community/discussions/27367) and [GitHub Skills](https://skills.github.com/).

**Reply by Spartiate91** [reply]
Thanks for you're informations 😉

**Comment by nyx437** [comment]
Hey Spartiate91 I'm all nooby too.  I like to encrypt things and I enjoy contemplating prime numbers.

**Reply by DSDmark** [reply]
And we are in the same boat. I'm noob too.

**Comment by DSDmark** [comment]
Hey @Spartiate91,

>  Nice to meet you and welcome to this great community, we are here for helping each other and try to make open source better.
This is the discussions section, if you want or need anything you can ask here, we try to help.

Also, you can check this out:-

>  Welcome to the GitHub Community on Discussions, Happy to have you here 
To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204). To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our GitHub Skill up exercises, and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912).
